data/worlds/classic.world

```diff
@@ -89,6 +89,7 @@ overlay
   ( overlay
     [ {grass}
     , mask (hash % 20 == 10) {grass, cotton}
+    , mask (hash % 20 == 10) {grass, wheat}
```

Comment:
Since both `cotton` and `wheat` use the same mask, one will completely override the other (I forget which).  Should use something like e.g. `hash % 20 == 11` for `wheat`.